recipe/jwt/recipeimplementation.go

```diff
@@ -30,11 +30,18 @@ func makeRecipeImplementation(querier supertokens.Querier, config jwtmodels.Type
 			payload = map[string]interface{}{}
 		}
 
+		shouldUseStaticSigningKey := true
+
+		if useStaticSigningKey != nil {
+			shouldUseStaticSigningKey = *useStaticSigningKey
+		}
+
 		response, err := querier.SendPostRequest("/recipe/jwt", map[string]interface{}{
-			"payload":    payload,
-			"validity":   validitySeconds,
-			"algorithm":  "RS256",
-			"jwksDomain": appInfo.APIDomain.GetAsStringDangerous(),
+			"payload":             payload,
+			"validity":            validitySeconds,
+			"algorithm":           "RS256",
+			"jwksDomain":          appInfo.APIDomain.GetAsStringDangerous(),
```

Comment:
should this also have apiBasePath?

Reply:
The api base path isnt really needed in this case. The `jwksDomain` is used as the `iss` claim in the JWT. Since the issuer is supposed to be used to simply identify who created the JWT, the api domain in this case is enough (even though we use a URL as the value, the URL isnt actually used in any way)

Its just that the request property is named badly (based on my understanding when we created the JWT recipe)